docker/development/Dockerfile

```diff
@@ -39,7 +39,7 @@ RUN cd /tmp \
  && git checkout tags/yaml-cpp-${YAML_CPP_VERSION} \
  && mkdir build \
  && cd build \
- && cmake -DCMAKE_POSITION_INDEPENDENT_CODE=TRUE -DYAML_BUILD_SHARED_LIBS=OFF .. \
+ && cmake -DCMAKE_POSITION_INDEPENDENT_CODE=TRUE -DYAML_BUILD_SHARED_LIBS=OFF -DYAML_CPP_BUILD_TESTS=OFF .. \
```

Comment:
Is this part of the debian change or just something you noticed?

Reply:
This was a change required since we were building tests in yaml-cpp with gtest 1.10 which doesn't work with latest cmake as it has some warnings that are treated as errors due to CMake flags. So i just switched off the test building, as we're not using the tests for a third party repo anyway.